src/NuGet.Clients/NuGet.PackageManagement.UI/DisplayVersion.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Globalization;
```

Comment:
> Unfortunately, this only applies to the Installed tab for now as the Browse tab doesn't have the deprecation metadata available when it renders the list.

Can you elaborate on why this won't work with browse?

Reply:
The list of versions that appears in this list is fetched from search in the Browse tab (which doesn't have deprecation metadata) and from registration in the Installed tab (which does), regardless of the fact that it ultimately fetches the full metadata from registration in both tabs.